recipe/migrations/osx_arm64.txt

```diff
@@ -1275,3 +1275,4 @@ puma
 grpcio-tools
 kwn-deform
 helix
+pcmsolver-split
```

Comment:
```suggestion
pcmsolver
```

Reply:
I think you want to specify the name of the packag,e and not the name of the recipe top level.

Reply:
I wasn't sure. `pcmsolver-split` is the feedstock name (yes, I made a mistake in not specifying `extra/feed-stockname` when the recipe got converted to multiout). The feedstock produces packages `pcmsolver` and `libpcm`.

Reply:
Either one will get the migrating started i think